### PR TITLE
react clock first write up

### PR DESCRIPTION
**Pull request by Kavster92**
[DEMO LINK](https://Kavster92.github.io/react_clock/)

### CODE REVIEW

**Review by Iksich — APPROVED** [review]
Looks good, one thing you can just, import React from 'react';
Instead of Component

File: src/App.tsx [hunk 1]
```diff
@@ -1,39 +1,64 @@
-import React from 'react';
 import './App.scss';
+import { Component } from 'react';
```
Comment (Iksich) [inline src/App.tsx:2]: import React from 'react';

**Review by Iksich — APPROVED** [review]

File: src/Clock.tsx [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+type State = {
+  today: string;
+};
+
+type Props = {
+  clockName: string,
+};
+
+const formatedDate = (): string => {
```
Comment (naziamoff) [inline src/Clock.tsx:11]: function should be named as a function, starting with a verb that describes what the function does
```suggestion
const getFormattedDate = (): string => {
```

File: src/Clock.tsx [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+  return new Date().toUTCString().slice(-12, -4);
+};
+
+export class Clock extends Component<Props, State> {
+  state = {
+    today: formatedDate(),
```
Comment (naziamoff) [inline src/Clock.tsx:17]: isn't this `now`?

**Review by naziamoff — APPROVED** [review]
Well done! Some minor comments

**Pull request closed without merging**